README.md

```diff
@@ -1,72 +1,72 @@
-# CMSIS Driver Validation
+# CMSIS-Driver Validation
 
-The branch *master* of this GitHub repository contains the CMSIS-Driver Validation. The [documentation](http://arm-software.github.io/CMSIS-Driver_Validation/html/index.html) is available under 
-http://arm-software.github.io/CMSIS-Driver_Validation/html/index.html
+Test suite for verifying that a peripheral driver implementation is compliant with the correposponding [CMSIS-Driver Specification](https://arm-software.github.io/CMSIS_5/Driver/html/index.html).
 
-Use [Issues](https://github.com/ARM-software/CMSIS-Driver_Validation#issues-and-labels) to provide feedback and report problems for CMSIS-Driver Validation.
+## Overview
```

Comment:
I suggest to emphasize main with **main** instead of *main*, because italics do not stand out at all.